### PR TITLE
Copy edit the concepts page.

### PR DESCRIPTION
**Pull request by jpeach**
This is a minor copy-editing pass over the concepts document. The
only changes should be minor grammatical, punctuation and formatting
improvements.

Signed-off-by: James Peach <jpeach@vmware.com>

### CODE REVIEW

**Comment by jpeach** [comment]
* needs mkdocs to run once the image is fixed #141

File: docs-src/concepts.md [hunk 1]
```diff
@@ -38,10 +38,10 @@ Gateway/Routes API will target the following personas:
 
 * **Infrastructure provider**: The infrastructure provider (infra) is
   responsible for the overall environment that the cluster(s) are operating in.
-  Examples include: the cloud provider (AWS, Azure, GCP, ...), the PaaS provider
-  in a company.
+  Examples include public cloud providers (AWS, Azure, GCP, ...), or PaaS providers
+  within an organization.
 * **Cluster operator**: The cluster operator (ops) is responsible for
-  administration of entire clusters. They manage policies, network access,
+  administration of entire clusters. They manage policies, network access and
```
Comment (robscott) [inline docs-src/concepts.md:44]: nit: I'd prefer a comma after network access.

Comment (jpeach) [reply docs-src/concepts.md]: Done.

**Comment by jpeach** [comment]
> * needs mkdocs to run once the image is fixed #141

Done.

**Comment by robscott** [comment]
Thanks!

/lgtm
/assign @bowei

**Comment by robscott** [comment]
@jpeach looks like this needs to be rebased now.
/ok-to-test

**Comment by jpeach** [comment]
> @jpeach looks like this needs to be rebased now.

Updated, thanks!

**Pull request merged**